apps/src/music/blockly/blocks/simple2.js

```diff
@@ -196,6 +198,36 @@ export const playPatternAtCurrentLocationSimple2 = {
     `
 };
 
+export const playChordAtCurrentLocationSimple2 = {
+  definition: {
+    type: BlockTypes.PLAY_CHORD_AT_CURRENT_LOCATION_SIMPLE2,
+    message0: 'play chord %1',
+    args0: [fieldChordDefinition],
+
```

Comment:
nit: I see `playPatternAtCurrentLocationSimple2` has an empty line here too, but not sure that it was deliberate :)

Reply:
Oh good catch, will fix